src/app.js

```diff
@@ -1 +1,45 @@
 'use strict';
+
+const { numberGenerator } = require('./numberGenerator');
+const { terminal } = require('./terminal');
+const { calculator } = require('./calculator');
+const { output } = require('./output');
+
+const { INPUT_MESSAGE, OUTPUT_MESSAGE } = require('./constants');
+
+const generatedNums = numberGenerator();
+
+const getNumber = (userInput) => {
+  if (userInput.length !== 4) {
+    terminal.question(
+      INPUT_MESSAGE.enterCorrectNumber,
+      num => getNumber(num)
+    );
+  } else if ((/0-9/).test(userInput)) {
```

Comment:
It is better to put this condition in a separate variable to improve readability